index.html

```diff
@@ -31,12 +31,145 @@ <h1 class="about-title">I’m Martin.Glad to see you!</h1>
+            <button class="see-3">See project</button>
+        </article>
+        <!-- CARD 4 -->
+        <article class="card-4">
+            <img src="image/Snapshoot Portfolio4.png" alt="image1" >
+            <h3 class="tonic-4">Multi-Post <br> Stories</h3>
+            <div class="canop-4">
+                <p class="can-info-4">CANOPY</p>
+                <div class="point-4"></div>
+                <p class="back-4">Back End Dev</p>
+                <div class="point-4"></div>
+                <p class="back-4">2015</p>
+            </div>
+            <p class="paranic-4">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="languages-4">
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">JavaScript</a></li>
+            </ul>
+            <button class="see-4">See project</button>
+        </article>
+    </section>
+    <!-- ABOUT SECTION -->
+    <section class="myself">
+        <h1 class="me">About Myself</h1>
+        <p class="para-detail">
+            Hello I'm a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+        <h3 class="conn-5">LET'S CONNECT</h3>
+        <ul class="social-5">
+            <li><a href="#"><img src="image/Vector (12).png" alt=""></a></li>
+            <li><a href="#"><img src="image/Linkedin icon.png" alt=""></a></li>
+            <li><a href="#"><img src="image/Vector (13).png" alt=""></a></li>
+            <li><a href="#"><img src="image/Vector (15).png" alt=""></a></li>
+            <li><a href="#"><img src="image/Vector (11).png" alt=""></a></li>
+        </ul>
+        <button class="btn-5">Get my resume</button>
+        <ul class="language">
+            <li class="lang">
+                <p>LANGUAGES</p>
+                <img src="image/Icon - Chevron - Down - Larger.png"  alt="">
+            </li>
+            <li class="java">
+                <img src="image/Ellipse 1.png" alt="">
+                <p>JavaScript</p>
+            </li>
+            <li class="htmll">
+                <img src="image/Ellipse 2.png" alt="">
+                <p>HTML</p>
+            </li>
+            <li class="csss">
+                <img src="image/Ellipse 3.png" alt="">
+                <p>CSS</p>
+            </li>
+            <li class="frameworks">
+                <p>FRAMEWORKS</p>
+                <img src="image/Icon - Chevron - Right - Larger.png" alt="">
+            </li>
+            <div class="line"></div>
+            <li class="skills">
+                <p>Skills</p>
+                <img src="image/Icon - Chevron - Right - Larger.png" alt="">
+            </li>
+            <div class="line"></div>
+        </ul>
```

Comment:
As requested by the previous reviewer and according to the [project requirements ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)

The technology in each category(e.g., "JavasScript", "HTML" and "CSS") is also a list `ul` and each of them a `li` 🤓👍

Example
```HTML
<li class="lang">
    <p>LANGUAGES</p>
    <img src="image/Icon - Chevron - Down - Larger.png"  alt="">
    
    <ul>
        <li class="java">
            <img src="image/Ellipse 1.png" alt="">
            <p>JavaScript</p>
        </li>
        <li class="htmll">
            <img src="image/Ellipse 2.png" alt="">
            <p>HTML</p>
        </li>
        <li class="csss">
            <img src="image/Ellipse 3.png" alt="">
            <p>CSS</p>
        </li>
    </ul>

</li>
```